sdk/tests/conformance/extensions/ext-disjoint-timer-query.html

```diff
@@ -58,6 +62,15 @@
         testPassed("No EXT_disjoint_timer_query support -- this is legal");
     } else {
         runSanityTests();
+
+        // Clear disjoint value.
+        gl.getParameter(ext.GPU_DISJOINT_EXT);
+
+        runElapsedTimeTest();
+        runTimeStampTest();
+
+        gl.finish();
+        setTimeout(checkQueryResults, 0);
```

Comment:
Ok, I addressed these issues in the latest patch. Also I have added a bunch of tests testing the conditions involving getting the current query.